microcosm_flask/audit.py

```diff
@@ -353,10 +358,11 @@ def parse_response(response):
 
 
 @defaults(
-    include_request_body=DEFAULT_INCLUDE_REQUEST_BODY,
-    include_response_body=DEFAULT_INCLUDE_RESPONSE_BODY,
-    include_path="true",
-    include_query_string="true",
+    include_request_body=typed(type=int, default_value=DEFAULT_INCLUDE_REQUEST_BODY),
```

Comment:
This has been bothering me for a while. We're manually doing various conversions farther down, but now we have code in microcosm to do this for us.